admin-dev/themes/new-theme/js/pages/order-states/form.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * 2007-2019 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+import initColorPickers from '@app/utils/colorpicker';
+import TranslatableChoice from '@components/form/translatable-choice';
+import TranslatableInput from '@components/translatable-input';
+import FormMap from '@pages/order-states/form-map';
+
+const {$} = window;
+
+$(() => {
+  initColorPickers();
+  new TranslatableInput();
+  new TranslatableChoice();
+
+  let templatePreviewWindow = null;
+  function viewTemplates($uri) {
+    if (templatePreviewWindow != null && !templatePreviewWindow.closed) {
+      templatePreviewWindow.close();
+    }
+    templatePreviewWindow = window.open(
+      $uri,
+      'tpl_viewing',
+      'toolbar=0,'
+      + 'location=0,'
```

Comment:
Never seen this way of writing long parameter string 😛 

It's funny and it works so 👍 ^^

Reply:
I'm personally not a fan, I would rather have it like this:

```js
templatePreviewWindow = window.open(
  $uri,
  'tpl_viewing',
  [
    'toolbar=0',
    'location=0',
    // ...
  ].join(',')
)
```

This way parameters and string members are visibly distinct. But it's not a big deal.

Reply:
Ping @sowbiba